lib/ansible/parsing/vault/__init__.py

```diff
@@ -143,48 +164,41 @@ def __init__(self, password):
         self.cipher_name = None
         self.b_version = b'1.1'
 
-    # really b_data, but for compat
-    def is_encrypted(self, data):
+    @staticmethod
+    def is_encrypted(data):
```

Comment:
I think these should stay regular methods. That way if VaultLib.is_encrypted() could be more
specific than the module scope version. The primary case I'm thinking of is checking if the data
is encrypted by the key that the VaultLib instance knows about. ie, 'is this file something _I_ can decrypt'

(granted at the moment, nothing uses that so it doesn't matter...)